### PR TITLE
Remove private frontend for Looker

### PR DESCRIPTION
**Pull request by chaodaiG**
There is nothing left running on it now

cc @2nd-half

### CODE REVIEW

File: prow/oss/cluster/cluster.yaml [hunk 1]
```diff
@@ -90,16 +90,6 @@ spec:
             name: hook
             port:
               number: 8888
-  - host: oss-prow-private.knative.dev
```
Comment (cjwagner) [inline prow/oss/cluster/cluster.yaml:93]: We can also remove the managed certs and associated ingress annotations too.

**Review by cjwagner — APPROVED** [review]
/hold

File: prow/oss/cluster/cluster.yaml (hunk 1 above)
Comment (chaodaiG) [reply prow/oss/cluster/cluster.yaml:93]: good point, done, both

**Review by cjwagner — APPROVED** [review]

**Comment by chaodaiG** [comment]
/unhold
other than updating ingress to not include private cert, this PR itself is pretty much noop.

will need to manually clean up the resources removed from this PR

**Pull request merged**